test/Toggle/GetActivatedTogglesTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Vovaxs
+ * Date: 24/08/2015
+ * Time: 10:31
+ */
+
+namespace Clearbooks\Labs\Toggle;
+
+
+use Clearbooks\Labs\Toggle\Entity\ActivatedToggleStub;
+use Clearbooks\Labs\Toggle\Gateway\ActivatedToggleGatewayDummy;
+use Clearbooks\Labs\Toggle\Gateway\ActivatedToggleGatewayStub;
+
+class GetActivatedTogglesTest extends \PHPUnit_Framework_TestCase
+{
+
+    public function setUp()
+    {
+
+    }
```

Comment:
Also I think you need to check the code review rota for this week @VolodymyrSergeyev as dan is my reviewer this week, don't want to be overloading him with PRs :smile: